Makefile.am

```diff
@@ -58,7 +58,11 @@ install-data-hook:
 hanoarchdir = $(datadir)/@PACKAGE@/hb_report
 hanoarch_DATA = hb_report/constants.py hb_report/utillib.py
 hanoarch_SCRIPTS = hb_report/hb_report
-EXTRA_DIST = $(hanoarch_DATA)
+preflightnoarchdir = $(datadir)/@PACKAGE@/preflight_check
```

Comment:
Strictly speaking, binary should not put in `datadir`.

Reply:
We can change this when we decide where to put it

Reply:
sure.